src/components/Home.js

```diff
@@ -24,9 +44,15 @@ const Home = () => {
       <button onClick={createToken}>Create a new list</button>
       <p>-or-</p>
       <p>Join an existing shopping list by entering a three word token.</p>
-      <form>
+      <form onSubmit={verifyToken}>
         <label htmlFor="token">Share token</label>
-        <input type="text" name="token" />
+        <input
+          type="text"
+          name="token"
+          value={sharedToken}
```

Comment:
Ahh, Megan told me this previously, but for the htmlFor, it's associated with an "id" property. So how about we have also have an "id" property on our `input` tag set to "token" to connect the `label` with the `input` ?

Reply:
That's a great point, Sandy! Totally forgot to add the `id` attribute to the input field. Will do that now. Thanks for suggesting it! 🙏